src/App.tsx

```diff
@@ -1,7 +1,42 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  wasPresed: boolean;
+  keyPres: null | string;
+};
+
+export class App extends React.Component<{}, State> {
+  state = {
+    wasPresed: false,
+    keyPres: null,
+  };
+
+  componentDidMount(): void {
```

Comment:
It's extra, bc TypeScript already knows about React types

```suggestion
  componentDidMount() {
```